### PR TITLE
Raven init

### PR DESCRIPTION
**Pull request by mehtaamish89**
For: https://github.com/Medology/cordova-hotspot-plugin/issues/1

- Removed the type definition files that caused the app the compilation to fail.
- Removed the line that was causing the failures.

### CODE REVIEW

**Review by goodevilgenius — APPROVED** [review]

File: src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  The MIT License (MIT)
- Copyright (c) 2016 Martin Reinhardt
+ Copyright (c) 2018 Raven
```
Comment (Chekote) [inline src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java]: You cannot remove the original copyright owner from this code, you can only add your copyright claim for your own additions/modifications:

```
Original work Copyright (c) 2016 Martin Reinhardt
Modified work Copyright (c) 2018 Raven
```

**Review by Chekote — CHANGES_REQUESTED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Martin Reinhardt
+Copyright (c) 2018 Raven
```
Comment (Chekote) [inline LICENSE]: You cannot remove the original copyright owner from this code, you can only add your copyright claim for your own additions/modifications:

```
Original work Copyright (c) 2016 Martin Reinhardt
Modified work Copyright (c) 2018 Raven
```

**Review by Chekote — CHANGES_REQUESTED** [review]

File: src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java (hunk 1 above)
Comment (mehtaamish89) [reply src/android/de/martinreinhardt/cordova/plugins/hotspot/HotSpotPlugin.java]: 👍Done, Was not aware of this.

File: LICENSE (hunk 2 above)
Comment (mehtaamish89) [reply LICENSE]: 👍 Done

**Review by joshmedeski — APPROVED** [review]
No type definitions, yuck. I will approve this PR so the project can move forward, but I'd like to add the type definitions back in the near future if possible.

Can you create a spike to reverse this type definition deletion?

**Pull request merged**